### PR TITLE
Assorted cleanups

### PR DESCRIPTION
**Pull request by xeno-by**

### CODE REVIEW

File: tests/jvm/src/test/resources/metacp.expect [hunk 1]
```diff
@@ -38,26 +67,36 @@ Schema => SemanticDB v3
...
+_root_.example. => package example
 _root_.example.Acc# => class Acc.{+8 decls}
-  extends AnyRef
-_root_.example.Acc#`<init>`(). => primaryctor <init>: (): Acc
+  extends scala.this.AnyRef
+_root_.example.Acc#`<init>`(). => primaryctor <init>: (): example.this.Acc
+  example => _root_.example.
   Acc => _root_.example.Acc#
-_root_.example.Acc#m1(). => private def m1: : Nothing
+_root_.example.Acc#m1(). => private def m1: : scala.this.Nothing
```
Comment (olafurpg) [inline tests/jvm/src/test/resources/metacp.expect]: Is this change necessary? I prefer the old type

Comment (olafurpg) [reply tests/jvm/src/test/resources/metacp.expect]: `scala.this.Int` is not even valid unless `scala` is an enclosing class

Comment (xeno-by) [reply tests/jvm/src/test/resources/metacp.expect]: See https://github.com/scalameta/scalameta/issues/1333 for discussion.
